docs/ECCTOOL.md

```diff
@@ -16,11 +20,262 @@ The following subcommands are available:
 | `ecctool stop`        | Stop ecChronos service                   |
 | `ecctool status`      | Show status of ecChronos service         |
 
-### Flags
+For more information about each subcommand refer to the specific sections below.
+
+### repairs
+
+`ecctool repairs` subcommand is used to show the status of all manual repairs.
+This subcommand has no mandatory parameters.
 
-Each of the subcommands support flags. For more information provide `-h` flag after subcommand.
-For example:
+#### Example
 
+In this example, we will use `ecctool repairs` to check the status of manual repairs.
+The output shows all manual repairs for all ecChronos instances.
+
+```bash
+---------------------------------------------------------------------------------------------------------------------------------------------------
+| Id                                   | Host Id                              | Keyspace | Table  | Status    | Repaired(%) | Completed at        |
+---------------------------------------------------------------------------------------------------------------------------------------------------
+| f4fb2b38-b9d0-4390-97ca-eeb284391f80 | ee32d9c7-1a4e-40c2-9b28-1000544011ae | test     | table1 | IN_QUEUE  | 0.00        | -                   |
+| f4fb2b38-b9d0-4390-97ca-eeb284391f80 | ba7665b2-5a7b-42b5-9f38-037f2da1e80a | test     | table1 | COMPLETED | 100.00      | 2022-09-22 13:40:07 |
+---------------------------------------------------------------------------------------------------------------------------------------------------
+Summary: 1 completed, 1 in queue, 0 blocked, 0 warning, 0 error
 ```
-ecctool repairs -h
-```
\ No newline at end of file
+
+Looking at the example output above, the columns are:
+
+`Id` - the manual repair ID, manual repair triggered on several hosts will have the same ID.
+
+`Host Id` - the host id of the Cassandra instance ecChronos responsible for performing manual repair is connected to.
+
+`Keyspace` - the keyspace the manual repair is run on.
+
+`Table` - the table the manual repair is run on.
+
+`Status` - the status of the manual repair.
+The possible statuses are:
+* `IN_QUEUE` - the manual repair is awaiting execution or is currently running
+* `ERROR` - the manual repair failed
+* `COMPLETED` - the manual repair is completed
+
+`Repaired(%)` - the number of ranges repaired vs total ranges.
```

Comment:
to avoid confusion, specify more specifically:

For the repair command, this value should never go down.
Or
Unlike Schedules, this value should never go down.